kvmagent/kvmagent/test/localstorage_testsuite/test_data_volume_1.py

```
@@ -0,0 +1,70 @@
+        _, virtio_scsi1_vol = vm_utils.attach_virtio_scsi_volume_to_vm(vm_uuid, virtio_scsi1_vol_uuid, virtio_scsi1_vol_path)
+
+        _, virtio2_vol = vm_utils.attach_virtio_volume_to_vm(vm_uuid, virtio2_vol_uuid, virtio2_vol_path)
+        _, virtio_scsi2_vol = vm_utils.attach_virtio_scsi_volume_to_vm(vm_uuid, virtio_scsi2_vol_uuid, virtio_scsi2_vol_path)
+
+        rsp = vm_utils.check_volume(vm_uuid, [virtio1_vol, virtio_scsi1_vol,virtio2_vol,virtio_scsi2_vol])
+        self.assertTrue(rsp.success)
+
+        xml = vm_utils.get_vm_xmlobject_from_virsh_dump(vm_uuid)
+        disk = xml.devices.disk[0]
+        vol_path = disk.source.file_
+        vol_uuid = os.path.basename(vol_path).split('.')[0]
```

Ruff (0.12.2)

`os` may be undefined, or defined from star imports

(F405)